readthedocs/notifications/storages.py

```diff
@@ -59,3 +77,78 @@ def add(self, level, message, extra_tags='', *args, **kwargs):  # noqa
                 return
         super(FallbackUniqueStorage, self).add(level, message, extra_tags,
                                                *args, **kwargs)
+
+
+class NonPersistentNotification(PersistentStorage):
+
+    """
+    Save one time (non-pesistent) messages in the database.
+
+    Messages are saved into the database. ``user`` object is mandatory but
+    ``request`` is needed.
+    """
+
+    # Non-persistent level numbers start at 100 and it's used to check if it's
+    # an actionable message or not
+    level = 100
```

Comment:
Is this not used?

Reply:
This is used from django-extends-messages: https://github.com/AliLozano/django-messages-extends/blob/master/messages_extends/storages.py#L104

The custom storage that I defined will ignore all messages that smaller than `100`.